.gitignore

```
@@ -21,7 +21,7 @@ packages/subgraph/tests/.bin
 .pnp.*
 coverage.json
 
-packages/contracts/plugin-info.json
+plugin-info.json
```

```suggestion
plugin-info-testing.json
```
This one should be ignored,